hedgehog/src/Hedgehog/Internal/Range.hs

```diff
@@ -210,7 +211,11 @@ linear x y =
 --   >>> bounds 99 $ linearFrom 0 (-10) 20
 --   (-10,20)
 --
-linearFrom :: Integral a => a -> a -> a -> Range a
+linearFrom :: Integral a
+  => a -- ^ Value to shrink toward (the value produced when the size parameter is 0)
+  -> a -- ^ Lower bound (the bottom of the range when the size parameter is 99)
+  -> a -- ^ Upper bound (the top of the range when the size parameter is 99)
+  -> Range a
```

Comment:
😅 Nitpick: Most (if not all?) comments in this codebase end with a `.`— It'd be nice if we do the same also here?

Reply:
sure, i can fix all that (give me a few days).  anything else?

Reply:
ah, travis.  i'll look into that, too...